packages/safe-apps-react-sdk/package.json

```diff
@@ -5,10 +5,6 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "license": "MIT",
-  "files": [
```

Comment:
If only "src" folder is necessary I would prefer only this to be added. Not all packages have tests, but if we add them we would also be adding them to the npm package making it grow in size without need.

Appart from adding source I would also add CHANGELOG and LICENSE if they exist

Reply:
updated senor